core/common/src/main/java/alluxio/network/RejectingServer.java

```diff
@@ -31,13 +32,23 @@ public final class RejectingServer extends Thread {
 
   private final InetSocketAddress mAddress;
   private ServerSocket mServerSocket;
+  private final long mSleepTime;
 
   /**
    * @param address the socket address to reject requests on
    */
   public RejectingServer(InetSocketAddress address) {
+    this(address, 0);
+  }
+
+  /**
+   * @param address the socket address to reject requests on
+   * @param sleepTime sleep time before close connection
+   */
+  public RejectingServer(InetSocketAddress address, long sleepTime) {
```

Comment:
is this only used in testing? if yes, add an VisibleForTesting annotation

Reply:
fix